tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -0,0 +1,185 @@
+from itertools import combinations
+
+import json
+import pytest
+from unittest import mock
+
+from mlflow.entities.model_registry import RegisteredModelTag
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_uc_registry_messages_pb2 import (
+    CreateRegisteredModelRequest,
+    UpdateRegisteredModelRequest,
+    DeleteRegisteredModelRequest,
+    GetRegisteredModelRequest,
+    SearchRegisteredModelsRequest,
+)
+from mlflow.store._unity_catalog.registry.rest_store import UcModelRegistryStore
+from mlflow.utils.proto_json_utils import message_to_json
+from mlflow.utils.rest_utils import MlflowHostCreds
+from tests.helper_functions import mock_http_200
+
+
+def host_creds():
+    return MlflowHostCreds("https://hello")
+
+
+@pytest.fixture
+def store():
+    return UcModelRegistryStore(host_creds)
+
+
+def _args(endpoint, method, json_body):
+    res = {
+        "host_creds": host_creds(),
+        "endpoint": "/api/2.0/mlflow/unity-catalog/%s" % endpoint,
```

Comment:
```suggestion
        "endpoint": f"/api/2.0/mlflow/unity-catalog/{endpoint}",
```

Can we use f-string?